recipes/stlearn/meta.yaml

```
@@ -0,0 +1,51 @@
+{% set name = "stlearn" %}
+{% set version = "0.4.4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/stlearn-{{ version }}.tar.gz
+  sha256: 77c2cf3bae2445daed8c377569f3c232d6ec632d09da2f326d2e3619ee873965
+
+build:
+  entry_points:
+    - stlearn=stlearn.app.app:main
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python >=3.7
+  run:
+    - bokeh ==2.4.2
+    - numba
+    - numpy ==1.21.5
+    - pillow ==9.0.1
+    - python >=3.7
+    - scanpy ==1.8.2
+    - scikit-image ==0.19.2
+    - tensorflow
+    - leidenalg
+    - louvain
+    - xlrd
+
+
+test:
+  imports:
+    - stlearn
+  requires:
```

Are you missing a pip check?
```suggestion
  commands:
    - pip check
  requires:
```